### PR TITLE
Process "Is this bot interactive?" question from the form

### PR DESCRIPTION
**Pull request by hugovk**
Process "Is this bot interactive?" column, and adds "interactive" tag if the answer is "Yes".

Otherwise no extra tag is added.

Also refactored the magic added-column numbers, if new questions are added in the future, just update this constant:

``` python
ADDED_COL = "M"
```

Resolves https://github.com/botwiki/botwiki.org/issues/64.

### CODE REVIEW

**Comment by stefanbohacek** [comment]
Awesome, thank you very much @hugovk !

File: content/bots/twitterbots/_sys_bot_.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Author: botsheeter.py
+Date: October 26, 2015
+Tags: twitter,twitterbot,active,python,rhyming,rhyme,band_names,open source,opensource,python,swizzard
+Nav: hidden
+Robots: index,follow
+*/
+
+[![](/content/bots/twitterbots/images/_sys_bot_.png)](https://twitter.com/_sys_bot_)
+
+[@_sys_bot_](https://twitter.com/_sys_bot_) is an [open source](https://github.com/swizzard/sys_bot) Twitter bot created by [swizzard](https://twitter.com/http://twitter.com/swizzard).
```
Comment (stefanbohacek) [inline content/bots/twitterbots/_sys_bot_.md:13]: @hugovk Ah, do you think you'd have time to look at this?

```
 [swizzard](https://twitter.com/http://twitter.com/swizzard).
```

I mean in botsheeter.py. If not, no worries, should be easy for me to fix later.

**Comment by hugovk** [comment]
Yep, I'll check.

**Comment by stefanbohacek** [comment]
@hugovk Thanks a lot!

**Pull request merged**